docs/proto/proto-docs.md

```diff
@@ -1148,7 +1148,7 @@ Query/ContractsByCode RPC method
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| `contracts` | [string](#string) | repeated | contracts are a set of contract addresses |
+| `contracts` | [string](#string) | repeated | contracts are a set of contract addresses return in the order of instantiation |
```

Comment:
need a break line or period before "return".
And, isn't it the order of "instantiation or migration"?

Reply:
> need a break line or period before "return".

I attempted to create a line break using a `<br>` tag, which resolved the issue here. However, in [query.pb.go](https://github.com/Finschia/wasmd/pull/103/files#diff-c5afb1f23df96451f326680e605bd104db59c63e88c5480a634f61610e627334)  at line 273, it became 
`// contracts are a set of contract addresses<br>`

So I will try to make a period before